magellan-journey-sample/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,5 @@
   <string name="detail_memo_text">This is detailed memo of your journey!!</string>
   <string name="learn_more_text">Here you will learn more about the expedition!</string>
   <string name="icon">icon</string>
+  <string name="activities">Activities</string>
```

Comment:
Nit: we probably shouldn't call this activities, since it conflicts with android's `Activity`. Maybe `Notifications` or `Alerts` or something?